generators/app/templates/package.json

```diff
@@ -75,10 +75,9 @@
     "lint-js": "eslint --ext .js,.jsx src",
     "lint-scss": "stylelint src/**/*.scss",
     "pretest": "yarn lint",
-    "prepublish": "yarn build",
-    "release:major": "yarn test && yarn version major -m \"Released version %s\" && yarn publish && git push --follow-tags",
-    "release:minor": "yarn test && yarn version minor -m \"Released version %s\" && yarn publish && git push --follow-tags",
-    "release:patch": "yarn test && yarn version patch -m \"Released version %s\" && yarn publish && git push --follow-tags",
+    "release:major": "yarn test && yarn build && npm version major -m \"Released version %s\" && npm publish && git push --follow-tags",
```

Comment:
@ryanthemanuel  informed me that in-fact the command are different. They're using different npm version commands. It would still be nice to share what is common but i won't press the point.